ktor-server/ktor-server-plugins/ktor-server-content-negotiation/jvmAndNix/src/io/ktor/server/plugins/contentnegotiation/RequestConverter.kt

```diff
@@ -18,7 +17,12 @@ internal fun PluginBuilder<ContentNegotiationConfig>.convertRequestBody() {
         val registrations = pluginConfig.registrations
         val requestedType = call.receiveType
 
-        if (requestedType.type in pluginConfig.ignoredTypes) return@onCallReceive
+        if (requestedType.type in pluginConfig.ignoredTypes) {
+            LOGGER.trace(
+                "[ContentNegotiation] Skipping for request type ${requestedType.type} because the type is ignored."
```

Comment:
what is the reason to have `[ContentNegotiation]` in every statement, while we have logger name ended with `ContentNegotiation`?

Reply:
Fixed